fake/src/lib.rs

```diff
@@ -198,12 +198,32 @@ pub trait Fake: Sized {
 }
 impl<T> Fake for T {}
 
+fn unique<U: Dummy<Faker> + PartialEq, R: Rng + ?Sized>(rng: &mut R, count: usize) -> Vec<U> {
+    let mut set = Vec::<U>::new();
+    while set.len() != count {
+        let new_item: U = Faker.fake_with_rng(rng);
```

Comment:
Not sure if it matters but I think this might be n^2 complexity, possibly creating the fully array, sorting and deduping it may be faster. Then appending to the array. But performance may not be an issue so feel free to ignore

Reply:
This has really bad performance.  Sorting would reduce the randomness.